### ISSUE TITLE
shutdown

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#37]
```
I don't think its been requested...
Possible to add a function to shutdown wii to off?

Currently both `home button` on remote, and `off` button (bottom right of
screen) do the same thing (regardless of option return to wii menu
(on/off), although I think preloader .30r4 may be messing with my wii))

Currently to turn off, I have to return to wii menu and run the wad `QUIT`.

Preloader starts wiiflow as installed file for me.

Basically can the option "return to wii menu (on/off)" be:
"on/off button  (wii menu/shutdown to off)
```

Original issue reported on code.google.com by `gould...@btopenworld.com` on 25 Oct 2009 at 11:06

**Comment by GoogleCodeExporter** [comment]
```
Same like: http://code.google.com/p/wiiflow/issues/detail?id=19

```

Original comment by `Naro...@googlemail.com` on 30 Oct 2009 at 5:09
- Changed state: **Duplicate**
